content/docs/cucumber/configuration.md

```diff
@@ -15,170 +15,376 @@ polyglot:
 # Type Registry
 
 {{% block "java,kotlin" %}}
-The type registry is used to configure parameter types and data table types. It can be configured by placing an implementation
-of `cucumber.api.TypeRegistryConfigurer` on the glue path.
+Parameter types let you convert parameters from cucumber-expressions to objects.
+Data table and doc string string types let you convert data tables and doc
```

Comment:
Repeat of "string"; is this intended or not? If so, maybe put `doc string` in back tics to indicate it is terminology.